django/db/models/constraints.py

```diff
@@ -18,18 +18,23 @@
 
 class BaseConstraint:
     default_violation_error_message = _("Constraint “%(name)s” is violated.")
+    violation_error_code = None
     violation_error_message = None
 
     # RemovedInDjango60Warning: When the deprecation ends, replace with:
-    # def __init__(self, *, name, violation_error_message=None):
-    def __init__(self, *args, name=None, violation_error_message=None):
+    # __init__(self, *, name, violation_error_code=None, violation_error_message=None):
+    def __init__(
+        self, *args, name=None, violation_error_code=None, violation_error_message=None
```

Comment:
We cannot add `violation_error_code` before `violation_error_message` because positional arguments are still supported.

Reply:
Positional arguments are handled by `*args` hence the order of keyword-only arguments after `*args* shouldn't have any impact.

Reply:
That's clever :+1: